### ISSUE TITLE
Include tensorflow-text 2.9.0 to support Ops for FastBertNormalize

### ISSUE THREAD

**Issue by dwyatte** [tensorflow/serving#2064]
## Bug Report
Marking as bug as from what I can tell, including support for tensorflow-text is the intention

### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: 5.10.124-linuxkit
- **TensorFlow Serving installed from (source or binary)**: Binary/Docker
- **TensorFlow Serving version**: 2.10

### Describe the problem
When `FastBertTokenizer` and submodules are included in SavedModel, TensorFlow Serving does not recognize them with the error

```
2022-10-04 19:09:35.657342: E external/org_tensorflow/tensorflow/core/grappler/optimizers/tfg_optimizer_hook.cc:134] tfg_optimizer{tfg-consolidate-attrs,tfg-toposort,tfg-shape-inference{graph-version=0},tfg-prepare-attrs-export} failed: INVALID_ARGUMENT: Unable to find OpDef for FastBertNormalize
	While importing function: __inference_call_182243
	when importing GraphDef to MLIR module in GrapplerHook
...
2022-10-04 19:10:22.730427: E external/org_tensorflow/tensorflow/core/grappler/optimizers/meta_optimizer.cc:954] function_optimizer failed: NOT_FOUND: Op type not registered 'FastBertNormalize' in binary running on 2d4c8711fa01. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) `tf.contrib.resampler` should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed.
2022-10-04 19:10:22.933758: E external/org_tensorflow/tensorflow/core/grappler/optimizers/tfg_optimizer_hook.cc:134] tfg_optimizer{tfg-consolidate-attrs,tfg-toposort,tfg-shape-inference{graph-version=0},tfg-prepare-attrs-export} failed: INVALID_ARGUMENT: Unable to find OpDef for FastBertNormalize
	While importing function: __inference_call_182243
	when importing GraphDef to MLIR module in GrapplerHook
``` 

### Exact Steps to Reproduce
```py
import tensorflow as tf
import tensorflow_text as text


class MyModel(tf.keras.Model):
    def __init__(self):
        super().__init__()
        self.tokenizer = text.FastBertTokenizer(vocab=["a", "b", "c", "[UNK]"])
        
    @tf.function(input_signature=[tf.TensorSpec([None], dtype=tf.string, name="text")])
    def call(self, inputs):
        return self.tokenizer.tokenize(inputs)
    

model = MyModel()
model(["hello world"])
model.save("tokenizer/0")
```

```sh
tensorflow_model_server --rest_api_port=8501 --model_base_path=$PWD/tokenizer
```

**Comment by broken** [comment]
IIRC, the update was rolled back because the ops were using c++17 features, but the model server was being compiled using an earlier version. I need to look back to see what the actual code is that broke it and update our patch file or see if tf.serving will upgrade their c++ version. Nobody was actively asking for the upgrade so I have been busy on other priorities that people were asking for which is why there hasn't been progress here.

I doubt I'll be able to get to it this week, but I'll try to find time next week since it is blocking you.

**Comment by piEsposito** [comment]
+1. Is there any workaround for that?

**Comment by singhniraj08** [comment]
@dwyatte,

Please let us know if the workaround provided in the above commit works for you. The workaround introduces a flag that lets you use tensorflow_text and BertTokenizer rather than FastBertTokenizer.

Thank you!

**Comment by dwyatte** [comment]
@singhniraj08 I wasn't using `transformers` for the use case that spawned this issue, although good to see they are adding the same functionality to make models and tokenizers servable end-to-end in TF Serving 🚀  

I do think we can change this to a feature request since it's a known limitation -- should I edit the original issue to reflect that or do you just want to change the label?

**Comment by broken** [comment]
@singhniraj08 The problem is that it is compiling with c++0x and not c++17 despite what is set in the .bazelrc file. Do you know somebody more familiar with tf.serving builds that can look at this?

**Comment by singhniraj08** [comment]
@SHvsMK, Could you please look into this feature request.

**Comment by broken** [comment]
A couple things I noticed. First, you have the flag `D_GLIBCXX_USE_CXX11_ABI=0` set with the comment that this is to bring parity with TF. TF flipped this recently, so I would recommend removing this as well from .bazelrc.

Another thing in the .bazelrc file I noticed was:

```
# TensorFlow Decision Forests does not use Absl StatusOr.
# Reason: The version of Absl included by TensorFlow lacks support for StatusOr.
build --define no_absl_statusor=1
```

.. but I don't believe this to be true. TF Text uses TF's version of absl and compiles fine using StatusOr. In fact, if you search the TF codebase you will notice uses of it there as well. Is there another reason for this? Could it be this compilation problem of not using C++17?

**Comment by singhniraj08** [comment]
@dwyatte, 

TF Text is updated to v2.9.0 in latest TF serving release [2.11.0](https://github.com/tensorflow/serving/releases/tag/2.11.0). Ref: [commit](https://github.com/tensorflow/serving/commit/e88506ab6709f2dc3a3bb1601ac4b84c23025611)

Please try the latest TF serving release 2.11.0 and let us know if you issue has been resolved. Thank you!

**Comment by dwyatte** [comment]
It looks like the changes didn't make the [TF Serving 2.11.0 release on Dockerhub](https://hub.docker.com/layers/tensorflow/serving/2.11.0-gpu/images/sha256-e813d6ea496dc03bac3bf65d2e7beda14783f67e29e77b7c1177ba90c1db971b?context=explore) but they are in the [nightly](https://hub.docker.com/layers/tensorflow/serving/nightly-gpu/images/sha256-e9d5e573a541b0bb4d55e28cdde4f71d3617938b73e83013ed436b3fb1d57558?context=explore)

I'm now noticing the SavedModel in the original example is invalid, but TF Serving nightly is no longer erroring when trying to load it, so I think we can close this issue. 

```
The given SavedModel SignatureDef contains the following input(s):
  inputs['text'] tensor_info:
      dtype: DT_STRING
      shape: (-1)
      name: serving_default_text:0
The given SavedModel SignatureDef contains the following output(s):
  outputs['output_1'] tensor_info:
      dtype: DT_INVALID
      shape: ()
      name: 
Method name is: tensorflow/serving/predict
```

Thanks @broken and @singhniraj08 for the continued support for TF Text in TF Serving, very handy for production NLP!
